db/db_impl.h

```diff
@@ -228,6 +228,9 @@ class DBImpl : public DB {
   using DB::Flush;
   virtual Status Flush(const FlushOptions& options,
                        ColumnFamilyHandle* column_family) override;
+  virtual Status Flush(
```

Comment:
It may be worth documenting that, even if users set `atomic_flush=true`, when calling manual flush they need to provide a list of all CFs in order to get atomicity.

Reply:
Sure, will do.